sdk/formrecognizer/ai-form-recognizer/src/models.ts

```diff
@@ -0,0 +1,984 @@
+  /**
+   * Subtotal field
+   */
+  Subtotal: NumberFieldValue;
+  /**
+   * Tax field
+   */
+  Tax: NumberFieldValue;
+  /**
+   * Tip field
+   */
+  Tip: NumberFieldValue;
+  /**
+   * Total field
+   */
+  Total: NumberFieldValue;
+  /**
+   * Transaction date field
+   */
+  TransactionDate: DateFieldValue;
+  /**
+   * Transaction time field
+   */
+  TransactionTime: TimeFieldValue;
+}
+
+/**
+ * Represents text values in a receipt
+ */
+export interface Receipt {
+  /**
+   * Receipt type, e.g., "Itemized"
+   */
+  receiptType: string;
+  /**
+   * Merchant name
+   */
+  merchantName?: string;
+  /**
+   * Merchant phone number
+   */
+  merchantPhoneNumber?: string;
+  /**
+   * Merchant address
+   */
+  merchantAddress?: string;
+  /**
+   * items in the receipt
+   */
+  items: ReceiptItem[];
+  /**
+   * Subtotal
+   */
+  subtotal?: number;
+  /**
+   * Tax
+   */
+  tax?: number;
+  /**
+   * Tip
+   */
+  tip?: number;
+  /**
+   * Total
+   */
+  total?: number;
+  /**
+   * Transaction date
+   */
+  transactionDate?: Date;
+  /**
+   * Transaction time
+   */
+  transactionTime?: string;
+}
+
+/**
+ * Represents an extracted receipt
+ */
+export interface RawReceiptResult {
+  /**
+   * Document type.
+   */
+  docType: "prebuilt:receipt";
+  /**
+   * First and last page number where the document is found.
+   */
+  pageRange: PageRange;
+  /**
+   * Dictionary of named field values.
+   */
+  fields: { [propertyName: string]: FieldValue };
+}
+
+/**
+ * Extracted receipt and values in it
+ */
+export type ExtractedReceipt = RawReceiptResult & Receipt;
+
+/**
+ * Raw texts extracted from a page in the input document.
+ */
+export interface RawExtractedPage {
+  /**
+   * The 1-based page number in the input document.
+   */
+  pageNumber: number;
+  /**
+   * The general orientation of the text in clockwise direction, measured in degrees between (-180,
+   * 180].
+   */
+  angle: number;
+  /**
+   * The width of the image/PDF in pixels/inches, respectively.
+   */
+  width: number;
+  /**
+   * The height of the image/PDF in pixels/inches, respectively.
+   */
+  height: number;
+  /**
+   * The unit used by the width, height and boundingBox properties. For images, the unit is
+   * "pixel". For PDF, the unit is "inch". Possible values include: 'pixel', 'inch'
+   */
+  unit: LengthUnit;
+  /**
+   * The detected language on the page overall. Possible values include: 'en', 'es'
+   */
+  // language?: Language;
+  /**
+   * When includeTextDetails is set to true, a list of recognized text lines. The maximum number of
+   * lines returned is 300 per page. The lines are sorted top to bottom, left to right, although in
+   * certain cases proximity is treated with higher priority. As the sorting order depends on the
+   * detected text, it may change across images and OCR version updates. Thus, business logic
+   * should be built upon the actual line location instead of order.
+   */
+  lines?: ExtractedLine[];
```

Comment:
Not sure if we need this here too, if it is included in all the fields?

Reply:
Will align with the final preview.1 design.